lib/sprockets/server.rb

```diff
@@ -23,7 +23,7 @@ def call(env)
       start_time = Time.now.to_f
       time_elapsed = lambda { ((Time.now.to_f - start_time) * 1000).to_i }
 
-      if env['REQUEST_METHOD'] != 'GET'
+      if !['GET', 'HEAD'].include?(env['REQUEST_METHOD'])
```

Comment:
Moving that line to a constant would take less time than writing this comment...
Without that change you are creating 3 objects upon each request. Which needs to be garbage collected later... This micro-optimization makes that particular place ~ 60% faster.